### PR TITLE
Ability to view comments

### PR DESCRIPTION
**Pull request by mattvv**
No styling, so someone can come in and style - but data is being put in, I'll likely add the ability to comment next!

### CODE REVIEW

File: components/Person.tsx
Comment (dervin10) [inline components/Person.tsx:13]: Yeah! optional chaining is that good 💯
```suggestion
    onClick={(): void => onClick?.('owners', person)}
```

File: components/Comments.tsx
Comment (dervin10) [inline components/Comments.tsx:8]: Since you assigned a default value, I think you don't need to double-check.

```suggestion
  comments: Comment[];
```

**Comment by adobito** [comment]
Nice! I don't see any issues but I'll wait until someone else approves it for merge. :)

File: components/Comments.tsx
Comment (dervin10) [inline components/Comments.tsx:24]: Here typescript will know it's a comment type, I know there's dev's that loves to put more explicit types into it, but I'm more in favor of, if typescript knows then fewer codes is enough. 😬

```suggestion
        (comment): JSX.Element => (
```

**Review by dervin10 — APPROVED** [review]
Look pretty good to me 💯

File: components/Person.tsx
Comment (mattvv) [reply components/Person.tsx:13]: I didn't think you could do optional chaining on method calls, can you?

File: components/Comments.tsx
Comment (mattvv) [reply components/Comments.tsx:24]: I don't mind being extra terse in some situations

**Pull request merged**
